composable_kernel/include/utility/number.hpp

```diff
@@ -40,5 +41,77 @@ __host__ __device__ constexpr auto operator%(Number<X>, Number<Y>)
     static_assert(Y > 0, "wrong!");
     return Number<X % Y>{};
 }
+
+template <long_index_t N>
+using Number64 = integral_constant<long_index_t, N>;
+
+template <typename Index0,
+          Index0 X,
+          typename Index1,
+          Index1 Y,
+          std::enable_if_t<std::is_same<Index0, long_index_t>::value ||
```

Comment:
Don't use std in kernel code unless absolutely necessary.

use enable_if_t and is_same defined in CK